src/full/Agda/Syntax/Translation/AbstractToConcrete.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE UndecidableInstances   #-}
+{-# LANGUAGE CPP                    #-}
```

Comment:
This should not add the `CPP` pragma here. But should probably be fixed in subsequent PR.